test/visualise.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
-import {renderToString} from "react-dom/server";
+const fs = require('fs');
+const { renderToString } = require('react-dom/server');
```

Comment:
Why the move to `require` away from `import`?

Reply:
I'm happy to revert this, but I've been wanting to discuss styled components with you, which was what prompted this change.

Reply:
It's not a big deal but it would be nice to be consistent.

Reply:
Agreed. Fixed.